sdk/core/Azure.Core.Experimental/src/JsonPatchOperationKind.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma warning disable 1591
```

Comment:
Hmm why was this needed?

Reply:
At one point types were public and I was lazy adding comments. Removed.